filechanges/boot/config/settings.ini

```diff
@@ -331,13 +330,42 @@ playlists_dir = /var/lib/mopidy/playlists
 data_dir = /var/lib/mopidy/localdata
 #This sets the method of indexing local files
 #You can set it to json or whoosh. Whoosh might be better with large collections, JSON is a bit more mature.
-# TODO: You cannot add settings for [local-sqlite]
-# because it breaks the startup script (won't read dashes in section names
 library = sqlite
 #library = whoosh
 #library = json
 
 scan_timeout = 1000
-scan_flush_threshold = 1000
+scan_flush_threshold = 100

```

Comment:
Yes the change to ```scan_flush_threshold = 100``` is intentional and to align with the configuration recommendations as listed at https://github.com/tkem/mopidy-local-sqlite